Security/TwoFactor/AuthenticationContext.php

```diff
@@ -108,4 +108,12 @@ public function setAuthenticated($authenticated)
     {
         $this->authenticated = $authenticated;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isAuthenticationTry()
+    {
+        return $this->request->request->has('_auth_code');
+    }
```

Comment:
"_auth_code" is just the default value. The parameter name can be configured, therefore must be passed from configuration.